src/test/java/net/minecraftforge/debug/client/TestScreen.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.debug.client;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.network.chat.Component;
+public class TestScreen extends Screen
+{
+    public static int open() {
+        Minecraft.getInstance().setScreen(new TestScreen(Component.translatable("screen.patch")));
+        return 1;
+    }
+
+    public TestScreen(Component pComponent) {
+        super(pComponent);
+    }
+
+    @Override
+    public void render(PoseStack pPoseStack, int pMouseX, int pMouseY, float pPartialTick) {
```

Comment:
Formatting: braces on newlines. Additionally, there shouldn't be a prefix for the parameters.
```suggestion
    public void render(PoseStack poseStack, int mouseX, int mouseY, float partialTick)
    {
```